.changelog/522.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:bug
+Fixed several missing/incorrect resource-level `project_id` implementations. NOTE: See associated PR for caveats on temporary regressions.
```

Comment:
I don't know that this entry provides sufficient context on the change. Can you summary in one or two sentences what exactly the change fixes?